Ninjathulhu/Ninjathulhu.Game/Sim.cs

```diff
@@ -1,14 +1,86 @@
-﻿
+﻿using System;
+using System.Collections.Generic;
+
 namespace Ninjathulhu.Game
 {
     public class Sim
     {
         public Map CurrentMap;
 
+        private Prefab PlayerPrefab;
+        private Prefab CultistPrefab;
+
         public Sim()
         {
+            DefinePrefabs();
```

Comment:
Nice! I love the fact that you've pushed the "game" part forward while I'm wanking around with event handlers.